web/frontend/src/index.tsx

```diff
@@ -226,8 +232,13 @@ const AppContainer = () => {
           isLogged: result.islogged,
           firstname: result.firstname,
           lastname: result.lastname,
-          role: result.role,
           authorization: result.islogged ? new Map(Object.entries(result.authorization)) : arr,
+          isAllowed: (authCtx: AuthState, subject: string, action: string) => {
+            return (
+              authCtx.authorization.has(subject) &&
```

Comment:
`authCtx` is the object this function is part of. So you shouldn't need it.

But arrow functions don't have access to this, so you need to declare it differently: 

```
isAllowed: function(subject: string, action: string) {
  return (
    this.authorization.has(subject) &&
    this.authorization.get(subject).indexOf(action) !== -1
 );
}
```

Reply:
Thanks for the help !